ci-operator/config/openshift/elasticsearch-operator/openshift-elasticsearch-operator-release-5.8.yaml

```diff
@@ -24,7 +24,7 @@ base_images:
     namespace: ocp
     tag: kube-rbac-proxy
   log-file-metric-exporter:
-    name: 5.y
+    name: "5.8"
```

Comment:
Do we want to keep building log-file-metric-exporter from main for 5.8 as well (similar to eventrouter)?